platform/core/Makefile

```diff
@@ -81,19 +80,19 @@ endif
 
 define build_for_architecture
 	rustup target add $(1)
-	cargo build \
+	cargo build -p ffi \
 		$(cargo_build_flag) \
 		--lib --target $(1)
 endef
 
 # ============ Android lib ============ #
 
-$(android_binaries): $(target) 
+$(android_binaries):
```

Comment:
Main difference here is that there's no conditional build, it will always be built, right?